modules/standard/Path.chpl

```diff
@@ -368,6 +368,34 @@ proc dirname(name: string): string {
   return dirname(path=name);
 }
 
+/*
+  Splits the given path into it's root and extension.
```

Comment:
Typo: "its" not "it's" ("possessive it" is a dumb English language corner case)

Reply:
Done